cfn-resources/search-deployment/test/README.md

```diff
@@ -0,0 +1,44 @@
+# MongoDB::Atlas::SearchDeployment
+
+## Impact 
+The following components use this resource and are potentially impacted by any changes. They should also be validated to ensure the changes do not cause a regression.
+ - SearchDeployment L1 CDK constructor
+
+
+## Prerequisites 
+### Resources needed to run the manual QA
+All resources are created as part of `cfn-testing-helper.sh`:
+
+- Atlas Project
+- Cluster
+
+## Manual QA
+Please follow the steps in [TESTING.md](../../../TESTING.md).
+
+
+### Success criteria when testing the resource
+1. After successful creation of the stack using a template from examples section, by seeing the details of the Cluster configuration in the Atlas UI (e.g. https://cloud.mongodb.com/v2/<project-id>#/clusters/detail/Cluster0) the isolated search nodes should appear together with the existing dedicated cluster nodes.
```

Comment:
```suggestion
1. After successful creation of the stack using a template from examples section, by seeing the details of the Cluster configuration in the Atlas UI (e.g. https://cloud.mongodb.com/v2/<project-id>#/clusters/detail/<clusterName>) the isolated search nodes should appear together with the existing dedicated cluster nodes.
```

Reply:
For consistency let's also include a screenshot here

Reply:
included